net/openvpn/patches/103-define-LN_serialNumber-for-wolfSSL.patch

```diff
@@ -0,0 +1,12 @@
+--- a/src/openvpn/ssl_verify_openssl.c
++++ b/src/openvpn/ssl_verify_openssl.c
```

Comment:
Same here. We don't know who authored it, what it does (yeah by looking to the source code, we know, but still). Would be nice to have backported the patch as it is e.g. with all the Git details like author, subject, commit description, etc.

This one do the stuff:
https://github.com/OpenVPN/openvpn/commit/028b501734b4a57dc53edb8b11a4b370f5b99e38.patch

Reply:
Same as above.